### DISCUSSION TITLE
Is there a way to "template" directories?

### DISCUSSION THREAD

**Discussion by colings86** [twpayne/chezmoi#2926] [Q&A] [answered]
I am wondering if there is a way to include/exclude a directory based on a variable. My use case here is that I only want to add `~/.config/nvim` to if I am on my work machine (I have a `.machine.purpose` variable which is set to `work` or `home` on different machines). I know that I can do this by templating all the files and wrapping the contents of the files in a template condition,  but I am wondering if there is a way to do it for the entire directory in one place rather than every file?

**Comment by twpayne** [comment, marked as answer]
Use a `.chezmoiignore` file:

```
{{ if ne .machine.purpose "work" }}
.config/nvim
.config/nvim/**
{{ end }}
```

**Reply by colings86** [reply]
Awesome. Thanks for that
